packages/blade/src/components/BaseHeaderFooter/BaseHeader.tsx

```diff
@@ -134,6 +153,7 @@ const _BaseHeader = ({
   metaComponentName,
 }: BaseHeaderProps): React.ReactElement => {
   const validatedTrailingComponent = useTrailingRestriction(trailing);
+  const shouldWrapTitle = titleSuffix && trailing && showBackButton && showCloseButton;
```

Comment:
What's happening here? Can we add a comment?

Reply:
Yup have added comment where it is being used https://github.com/razorpay/blade/pull/1298#discussion_r1244795207